### PR TITLE
Minecraft: Allow commands to be sent to minecraft server.

### PR DESCRIPTION
**Pull request by ibaton**
Minecraft: Allow commands to be sent to minecraft server.
Fixed problem with uninitialized player on startup.

Signed-off-by: Mattias Markehed <mattias.markehed@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/handler/MinecraftServerHandler.java [hunk 1]
```diff
@@ -117,10 +126,28 @@ private void updateServerState(ServerData serverData) {
...
+        if (connection != null) {
+            connection.sendMessage(message);
+            return true;
+        }
+        return false;
+    }
+
     @Override
     public void dispose() {
         logger.info("Disposing minecraft server thing");
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/handler/MinecraftServerHandler.java]: this should actually be debug level

File: addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/handler/MinecraftSignHandler.java [hunk 2]
```diff
@@ -114,7 +126,26 @@ public void updateState(String channelID, State state) {
...
+     * Send a sign command to server.
+     *
+     * @param type the type of command to send
+     * @param signName the sign that the command targets
+     * @param value the value related to command
+     */
+    private void sendSignCommand(String type, String signName, String value) {
+        SignCommandData signCommand = new SignCommandData(type, signName, value);
+        JsonElement serializedCommand = gson.toJsonTree(signCommand);
+        logger.debug("Command: " + serializedCommand);
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/handler/MinecraftSignHandler.java]: please use parameterized (not concatenating) logging

File: addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/message/data/commands/PlayerCommandData.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package org.openhab.binding.minecraft.message.data.commands;
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/message/data/commands/PlayerCommandData.java]: license header missing

File: addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/message/data/commands/SignCommandData.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
+package org.openhab.binding.minecraft.message.data.commands;
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/message/data/commands/SignCommandData.java]: license header missing

**Comment by ibaton** [comment]
@kaikreuzer 
I have addressed the issues above and updated the PR.

**Pull request merged**